.coderabbit.yaml

```diff
@@ -0,0 +1,81 @@
+# Docs: https://docs.coderabbit.ai/configure-coderabbit
+# Schema: https://coderabbit.ai/integrations/schema.v2.json
+
+language: en
+
+tone_instructions: |
+  Provide feedback in a professional, friendly, constructive, and concise tone.
+  Offer clear, specific suggestions and best practices to help enhance the code quality and promote learning.
+
+early_access: true
+
+knowledge_base:
+  # The scope of learnings to use for the knowledge base.
+  # `local` uses the repository's learnings,
+  # `global` uses the organization's learnings,
+  # `auto` uses repository's learnings for public repositories and organization's learnings for private repositories.
+  # Default value: `auto`
+  learnings:
+    scope: global
+  issues:
+    scope: global
+  pull_requests:
+    scope: global
+
+reviews:
```

Comment:
@gberenice one thing I want to see in here: Let's add `path_instructions` for all TF files and give it something like this: 
```
You're a terraform expert and have read all of the documentation at https://developer.hashicorp.com/terraform/docs. You have a strong grasp on the syntax and favor suggesting updates that you know are accurate. You are a big fan of the Cloud Posse / SweetOps ecosystem follow a lot of their best practices from https://docs.cloudposse.com/best-practices/terraform/. 
```

Reply:
This is just stuff off the top of my head and I'm honestly not even sure that sharing the links will work... But we should ask CR support if it will. The ideal is that we can share links like that in this file, but I imagine I'm asking too much. 

If we have specifics we think would be good to add here... then do so. I am just trying for ideal here so we don't need to write out too much. But we may need to resort to that.